ledfx/effects/audio.py

```diff
@@ -324,10 +353,10 @@ def open_audio_stream(device_idx):
             )
 
             self._stream.start()
+            self._audio_stream_active = True
 
         try:
             open_audio_stream(device_idx)
-            self._is_activated = True
         except OSError as e:
             _LOGGER.critical(
                 f"Unable to open Audio Device: {e} - please retry."
```

Comment:
This path leaves self._audio_stream_active = True but I don't think that is the case, if you are moving the True to be at the top, it should be set False here.

Reply:
oh, I see it handled in the deactivate